src/poli/core/registry.py

```diff
@@ -130,6 +130,12 @@ def register_isolated_function(
     force: bool = True,
     **kwargs,
 ):
+    if name == _OBSERVER:
+        warnings.warn(
+            "The name "
+            + name
+            + " is a reserved keyword. Please choose another name. Doing nothing."
+        )
```

Comment:
I think it might be better to raise an Exception! What do you think?

Reply:
Okay, will do.